components/organisms/o-payment.vue

```diff
@@ -116,6 +116,10 @@
         class="form__element"
         name="phone"
         :label="$t('Phone Number')"
+        :required="true"
+        :valid="!$v.payment.phoneNumber.$error"
+        :error-message="$t('Please provide valid phone number')"
```

Comment:
You should add it to theme's translations in resource/i18n

Reply:
I added this but I don't know how to test it?

Reply:
In your `config/local.json`:
- Set `multistore` to `true`
- In `de` section change index name from `vue_storefront_catalog_de` to `vue_storefront_catalog`
- Restart app, run `yarn dev` and open http://localhost:3000/de - translations should be in German (de-DE.csv)

Reply:
thanks, I tested and fixed them.